survey-runner.tf

```diff
@@ -110,10 +110,20 @@ module "survey-runner-vpc" {
   vpc_cidr_block = "${var.vpc_cidr_block}"
 }
 
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
+  EC2_INSTANCE_TAG = "RabbitMQ"
+  RETENTION_DAYS   = 10
```

Comment:
You need to use the variable here. e.g..
`RETENTION_DAYS   = ${var. ebs_snapshot_retention_days}`